frontend/src/modules/S3_Datasets/components/DatasetStartCrawlerModal.js

```diff
@@ -132,7 +132,7 @@ export const DatasetStartCrawlerModal = (props) => {
                         error={Boolean(touched.prefix && errors.prefix)}
                         fullWidth
                         helperText={touched.prefix && errors.prefix}
-                        label={`s3://${dataset.S3BucketName}/`}
+                        label={`s3://${dataset.restricted.S3BucketName}/`}
```

Comment:
nit: i know tab is limited to admin but any need to also protect this with `emptyPrintUnauthorized` or similar?